submodules/misc/views/set-ringtone.html

```diff
@@ -0,0 +1,14 @@
+<div class="dialog_popup callflows-port">
+	<h1>{{ i18n.oldCallflows.set_ringtone }}</h1>
+	<form name="form" method="post" action="#">
+		<div class="form_content">
+			<div class="popup_field">
+				<label for="set_ringtone_select">{{ i18n.oldCallflows.set_ringtone_select }}: </label>
```

Comment:
The `for` attribute needs to match the `id` attribute of the input or else clicking on it will not auto focus the input.
```suggestion
				<label for="alert_info">{{ i18n.oldCallflows.set_ringtone_select }}: </label>
```